src/nupic/engine/__init__.py

```diff
@@ -49,7 +49,7 @@
     ("nupic.regions.KNNClassifierRegion", "KNNClassifierRegion"),
     ("nupic.regions.PluggableEncoderSensor", "PluggableEncoderSensor"),
     ("nupic.regions.PyRegion", "PyRegion"),
-    ("nupic.regions.RecordSensor", "RecordSensor"),
```

Comment:
See other comments on a few of the places that need to be updated to `"py.RecordSensor"`. Can you change this back to my suggestion so I can see everything in context. I'm not sure which permutations you tried.